CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 
 ### Fixed
 - Public: Handle non JSON error responses and return a `Connection Lost` error to the user
+- Public: Make sure "full screen" mode is exited when navigating away from enlarged preview
```

Comment:
it's actually UI, since it's doesn't interfere with the sdk api (that's the case that makes it "public")

Reply:
Oh, thanks, will change that.